### PR TITLE
new devicetype dl-pr26

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/decentlab/PR-26/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    {
+      "properties": {
+        "pressure": {
+          "title": "Pressure",
+          "unit": "bar",
+          "description": "Barometric Pressure in bar",
+          "type": "number",
+          "minimum": -0.1,
+          "maximum": 1
```
Comment (mariusgiger) [inline types/decentlab/PR-26/default.schema.json:18]: are you sure about these?

File: types/decentlab/PR-26/uplink.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    return result;
+  }
+};
+
+function consume(event) {
+  var payload = event.data.payload_hex;
+  var sample = decentlab_decoder.decode(payload);
+
+  sample.pressure = sample["Pressure"]["value"];
+  //sample.pressureUnit = sample["Pressure"]["unit"];
```
Comment (mariusgiger) [inline types/decentlab/PR-26/uplink.js:81]: remove unused code

File: types/decentlab/PR-26/uplink.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  delete sample["Pressure"];
+
+  sample.temperature = sample["Temperature"]["value"];
+  //sample.temperatureUnit = sample["Temperature"]["unit"];
+  delete sample["Temperature"];
+  emit('sample', { "data": { voltage: sample["Battery voltage"]["value"], protocolVersion: sample.protocolVersion, deviceID: sample.deviceID } });
+  delete sample["protocolVersion"];
+  delete sample["deviceID"];
+  delete sample["Battery voltage"];
```
Comment (mariusgiger) [inline types/decentlab/PR-26/uplink.js:90]: why deletes?

**Review by mariusgiger — CHANGES_REQUESTED** [review]

File: types/decentlab/PR-26/default.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/decentlab/PR-26/default.schema.json:18]: Its more like 0 to 1 Bar but the example gives out a negative number
image
image

File: types/decentlab/PR-26/uplink.js (hunk 3 above)
Comment (J2thatsme) [reply types/decentlab/PR-26/uplink.js:90]: I use the decode sample for multiple outputs

Comment (mariusgiger) [reply types/decentlab/PR-26/uplink.js:90]: yes but there should be a better way than deleting the keys

File: types/decentlab/PR-26/default.schema.json (hunk 1 above)
Comment (mariusgiger) [reply types/decentlab/PR-26/default.schema.json:18]: ok

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**